sips/sip-030/sip-030-wallet-interface.md

```diff
@@ -0,0 +1,528 @@
+
+# Abstract
+
+This SIP proposes the integration of the standard into the Stacks blockchain's "Connect" and "Auth" system.
+The goal is to replace the current Connect interface, primarily used by web applications to connect with browser extensions and mobile apps, with a more straightforward protocol.
+The proposal consists mainly of standardizing JSON compatible interfaces for use with wallet interfaces.
+
+# Motivation
+
+The current Connect system, which has evolved since the Blockstack era, is primarily utilized by web applications for interfacing with wallets.
+However, many aspects of the existing "Connect" and "Auth" libraries are no longer required, leading to unnecessary complexity and brittleness in wallet connectivity.
+
+Recent attempts to standardize the interface have sparked valuable discussions but have not culminated in a ratified standard, largely due to the stable state of the existing system.
+This SIP aims to address these issues by adopting the WBIPs standards, which offer a more suitable RPC-style interface for modern web applications.
+The simplified protocol will allow integration without heavy dependencies (like Auth) and provider a more extendable interface for wallets.
+
+Additionally, this SIP is motivated by the increased traffic of Ordinal inscriptions on Bitcoin and the Stacks ecosystem growing closers to Bitcoin.
+The community has recognized the need for a more unified approach to wallet connectivity (e.g. Bitcoin and PSBTs for previously Stacks-only wallets).
+By adopting the new standard, we aim to align the community towards a common and modern protocol for wallet interaction in web applications.
+Importantly, the decision to use an existing standard (rather than designing a new one or reworking Auth) is intentional — to avoid further division or split ownership within the community.
+
+There was an attempt to re-use existing standards/protocols from other ecosystems via the WBIPs working group — but no consensus was found that was a perfect fit or had enough traction for the larger layer-2 ecosystem.
+So this SIP aims to capture the important features for the Stacks ecosystem, with a focus on extensibility.
+
+# Specification
+
+The proposed changes are listed as follows:
+
+Specify JSON-RPC 2.0 compatible methods and payloads for wallet interaction.
+These can be used via a browser object (i.e., via the `window.btc.request` method) or similar interfaces like WalletConnect.
+
+# Backwards Compatibility
+
+The implementation of this proposal is not necessarily backward compatible.
+However, wallets implementing the new standard are advised to maintain the previous system to support legacy applications during a transition period.
+Existing applications using the current Auth system should continue to operate, but immediate changes are recommended once this SIP is ratified.
+The Connect library should go live with the new standard, once most major wallets have adopted the new standard.
```

Comment:
I don't like the wording here, it opens up a lot of potential questions that aren't likely to be in this SIP. 

i think you can reword it to reflect a timeframe, rather than a nebulous "once most major wallets have adopted". 
otherwise, i think you'll have to enumerate which wallets will need to be updated, and how much time will be provided for that upgrade.

Reply:
True, I'll try to add it to the ratifying section